docs/libraries/typespec-client-generator-core/rules/property-name-conflict.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "property-name-conflict"
+---
+
+```text title="Full name"
+@azure-tools/typespec-client-generator-core/property-name-conflict
+```
+
+Verify that there isn't a name conflict between a property in the model, and the name of the model itself
+
+#### ❌ Incorrect
```

Comment:
I think we should use Invalid over Incorrect.

Reply:
Yeah just pattern from how eslint document its rules, e.g. https://typescript-eslint.io/rules/class-literal-property-style

But if we feel like changing then we can discuss the common format we want